Sources/Commands/SwiftTestTool.swift

```diff
@@ -199,6 +203,8 @@ public struct SwiftTestTool: SwiftTool {
                 opts.chdir = path
             case .skipBuild:
                 opts.buildTests = false
+            case .buildPath(let buildPath): ()
```

Comment:
It does compile but clearly that should not be there :)

I have corrected this.

-Kyle



> On Jun 30, 2016, at 11:11 AM, Ankit Agarwal <notifications@github.com> wrote:
> 
> In Sources/Commands/SwiftTestTool.swift <https://github.com/apple/swift-package-manager/pull/455#discussion_r69151059>:
> 
> > @@ -199,6 +203,8 @@ public struct SwiftTestTool: SwiftTool {
> >                  opts.chdir = path
> >              case .skipBuild:
> >                  opts.buildTests = false
> > +            case .buildPath(let buildPath): ()
> there is a trailing () in this line, does this compile o.o?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/apple/swift-package-manager/pull/455/files/24ac4ba9842dbf9b99732e8bb2c012cb24a8bca9#r69151059>, or mute the thread <https://github.com/notifications/unsubscribe/ABrJu86jKPX0E38j-YPx3PqxrSd-ETO6ks5qQ9yfgaJpZM4JCQco>.
>